build.sbt

```diff
@@ -70,6 +76,11 @@ javaOptions ++= Seq("-Xmx512m", "-Djava.awt.headless=true")
 
 scalacOptions ++= Seq("-deprecation", "-unchecked")
 
+testOptions in Test <+= (scalaVersion) map {
+  case v if v.startsWith("2.12") => Tests.Argument("-l", "disable_scala2.12")
```

Comment:
Wow, I am learning new things here :)
Can you please change the labels to something like `pre-scala2.12` and `scala2.12+`? I would like these names to still make sense when 2.13 comes out.